README.md

```diff
@@ -93,6 +93,31 @@ For example, on Fedora:
 
 - to add more python dependencies: add to `requirements.txt` and `requirements-workers.txt`
 
+## Database Migrations
+
+Follow steps below for data migrations.
+
+- Checkout the master branch with ensure no schema changes are present in `cachito/web/models.py`
+- Set `SQLALCHEMY_DATABASE_URI` to `sqlite:///cachito-migration.db` in `cachito/web/config.py` under the `Config` class
+- Run `cachito db upgrade` which will create an empty database in the root of your Git repository called `cachito-migration.db` with the current schema applied
+- In case of schema changes,
```

Comment:
You may want to mention that you should checkout your new branch just before this.